example/detox/e2e/test/fast_image/fast_image_cookie_token_api_client_request.e2e.js

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import {fastImageServerUrl} from '@support/test_config';
+import {
+    ApiClientFastImageScreen,
+    ApiClientScreen,
+} from '@support/ui/screen';
+import {verifyApiClient} from '../helpers';
+
+describe('Fast Image Cookie Token - API Client Request', () => {
+    const testBaseUrl = fastImageServerUrl;
+    const testLoginUrl = `${testBaseUrl}/login/123`;
+    const testImageUrl = `${testBaseUrl}/protected/api/files/fast-image.jpg`;
+    const testName = 'Fast Image Server API';
+    const {
+        fastImage,
+        imageNotSupportedIcon,
+        setImageUrl,
+    } = ApiClientFastImageScreen;
+
+    beforeAll(async () => {
+        await ApiClientScreen.open(testName);
+        await verifyApiClient(testName, testBaseUrl);
+        await ApiClientScreen.fastImageButton.tap();
+    });
+
+    it('should display fast image - with cookie token on protected request', async () => {
+        // # Login to get token
+        await setImageUrl(testLoginUrl);
+        await expect(imageNotSupportedIcon).toBeVisible();
+
+        // # Set image url
+        await setImageUrl(`${testImageUrl}?ignoreHeaderToken=true`);
```

Comment:
@migbot Thanks for the review! I improved it by using a `tokenSource=headers|query|cookies` parameter. I removed `ignoreCookieToken`, `ignoreHeaderToken`, and `ignoreQueryToken` but kept the general `ignoreToken` in case we want to add a test that would require it.

I updated the docs as well